plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -410,19 +401,41 @@ fu_ccgx_hpi_device_reg_read (FuCcgxHpiDevice *self,
 	g_autofree guint8 *bufhw = g_malloc0 (self->hpi_addrsz);
 	for (guint32 i = 0; i < self->hpi_addrsz; i++)
 		bufhw[i] = (guint8) (addr >> (8 * i));
-	if (!fu_ccgx_hpi_device_i2c_write (self, bufhw, self->hpi_addrsz,
-					   CY_I2C_DATA_CONFIG_NAK, error)) {
-		g_prefix_error (error, "write error: ");
-		return FALSE;
-	}
-	if (!fu_ccgx_hpi_device_i2c_read (self, buf, bufsz,
-					  CY_I2C_DATA_CONFIG_STOP | CY_I2C_DATA_CONFIG_NAK,
-					  error)) {
-		g_prefix_error (error, "read error: ");
-		return FALSE;
+	
+	for (gint retry = 1 ; retry <= HPI_CMD_RETRY_CNT ; retry++) {
+		g_autoptr(GError) error_local = NULL;
+		if ( retry > 1 ) {
```

Comment:
don't need whitespace before retry and after 1

Reply:
I will remove whitespaces.

Reply:
I fixed it.